app/assets/javascripts/discourse/app/lib/user-status-on-autocomplete.js

```diff
@@ -0,0 +1,36 @@
+import createUserStatusMessage from "discourse/lib/user-status-message";
+
+let tippyInstances = [];
```

Comment:
Cool that we put it here and there's no need to store it in composer components anymore 👍.